app/javascript/mastodon/features/compose/index.tsx

```diff
@@ -82,28 +77,39 @@ const Compose: React.FC<{ multiColumn: boolean }> = ({ multiColumn }) => {
     [dispatch],
   );
 
+  const scrollNavbarIntoView = useCallback(() => {
+    const navbar = document.querySelector('.navigation-panel');
+    const prefersReducedMotion = window.matchMedia(
+      '(prefers-reduced-motion: reduce)',
```

Comment:
We have `reduceMotion` exported from `mastodon/initial_state` which is manually set in preferences. Probably we should use that since that's used everywhere else?

Reply:
Perfect, I'll use that, thanks!